test/unit/logger/easy_logging_starter_test.js

```diff
@@ -8,7 +8,8 @@ const fsPromises = require('fs/promises');
 const path = require('path');
 const os = require('os');
 const Logger = require('../../../lib/logger');
-require('../../../lib/snowflake'); // import of it sets up node logger
+require('../../../lib/snowflake');
+const { exists } = require('../../../lib/util'); // import of it sets up node logger
```

Comment:
should the comment change line?

Reply:
I don't know why the comment changed this line. Fixed.